src/bun.js/node/node_fs.zig

```diff
@@ -3573,30 +2909,15 @@ pub const Arguments = struct {
             _ = self;
         }
 
-        pub fn fromJS(ctx: JSC.C.JSContextRef, arguments: *ArgumentsSlice, exception: JSC.C.ExceptionRef) ?FdataSync {
-            const fd = JSC.Node.fileDescriptorFromJS(ctx, arguments.next() orelse {
-                if (exception.* == null) {
-                    JSC.throwInvalidArguments(
-                        "File descriptor is required",
-                        .{},
-                        ctx,
-                        exception,
-                    );
-                }
-                return null;
-            }, exception) orelse {
-                if (exception.* == null) {
-                    JSC.throwInvalidArguments(
-                        "fd must be a number",
-                        .{},
-                        ctx,
-                        exception,
-                    );
-                }
-                return null;
+        pub fn fromJS(ctx: JSC.C.JSContextRef, arguments: *ArgumentsSlice) bun.JSError!FdataSync {
+            const fd = try JSC.Node.fileDescriptorFromJS(ctx, arguments.next() orelse {
+                ctx.throwInvalidArguments("file descriptor is required", .{});
+                return error.JSError;
+            }) orelse {
+                ctx.throwInvalidArguments("file descriptor must be a number", .{});
+                return error.JSError;
             };
-
-            if (exception.* != null) return null;
+            arguments.eat();
```

Comment:
is this fixing a bug?

Reply:
no because its the last argument but it was to keep consistency with the others